theano/tests/test_flake8.py

```diff
@@ -3,9 +3,14 @@
 """
 
 from nose.plugins.skip import SkipTest
-import os
+import os, sys
```

Comment:
Yes, but why scrutinizer didn't complain? That is the problem I have with this PR, it should have also failed! I must have a bug in my config. But I don't know where. @bartvm do you have an idea why it scrutinizer wouldn't have failed?